doc/guide/admin_slurm.rst

```diff
@@ -63,39 +91,46 @@ Flux has a solid security design
-  with full capability and configurability, and a :term:`step` is really
-  a :term:`job` in Flux.  Limitations of Slurm's step manager have been
-  a source of long standing problems.
+  In Slurm, a *job* is a resource allocation scheduled by the Slurm scheduler.
+  Within the allocation, a *job step* is a unit of work scheduled by the the
+  Slurm step manager. Since a new Flux instance can be launched as a
+  :term:`job` in another Flux instance (subdividing its resources), there is
+  no need have a step abstraction or a step manager.  All units of work are
+  jobs scheduled and launched by the same robust system.  Limitations of
+  Slurm's step manager have been a source of long standing problems.
 
 Flux APIs are not an afterthought
   Well thought out Python and C APIs make it easy to integrate Flux with
   workflow systems and new environments.
 
 Flux has a rich resource representation
   Flux's design incorporates a graph-based resource model that is
-  extensible to arbitrary resource types.  Caveat: not yet fully implemented.
+  extensible to arbitrary resource types.
 
 Flux is dynamic
-  Flux supports growing and shrinking allocations.  Caveat: not yet fully
-  implemented.
+  The Flux design supports growing and shrinking allocations.
 
 Flux is scalable
   Flux's recursive launch design enables each allocation to scale
   independently.  Each instance of Flux has job throughput and node count
   scalability comparable to Slurm.  But a cluster will typically be running
   many instances of Flux compared to one instance of Slurm.
 
-Flux encourages experimentation
-  In addition to extensibility via plugins (see :ref:`background_components`),
-  unprivileged users can trivially launch reconfigured or modified single-user
-  Flux instances in system resource allocations.
-
-Flux uses reactive messaging
+Flux uses event-driven messaging
   In contrast to Slurm's multi-threaded, monolithic server design, Flux
-  is built upon distributed message brokers and reactive (asynchronous)
-  agents that communicate only with messages.  Building distributed services
-  on this substrate is interesting, fun, and scales well.
+  is built upon distributed message brokers and event-driven (asynchronous,
+  reactive) agents that communicate only with messages.  Building distributed
+  services on this substrate is interesting, fun, and scales well.
+
+Flux encourages experimentation
+  A single-user Flux instance can be started standalone or within a resource
+  allocation on any system without administrator privileges.  Furthermore,
+  your Flux instance can be reconfigured, extended with plugins, or modified.
+  If you want to add a new Flux service, implement a new scheduler plugin,
+  wire your workflow system directly into Flux's cerebral cortex, or just
+  litter the code with print statements to understand it better, you are
+  encouraged to do so!  This freedom cannot be offered by monolithic systems
+  that run mostly as root.
 
```

Comment:
What about something about converged environments, or cloud? What I say in talks is something along the lines of how the dynamism / elasticity, event-driven architecture and scalability of Flux make it amenable to cloud setups, and we can (and have) deployed across clouds for Kubernetes, VMs, and as components in cloud services. "Flux doesn't care where you run it" essentially.

Reply:
Here is are two suggestions:

```console
Flux is agnostic to where you deploy it.

    Flux is modular and composable, making it easy to integrate into converged and cloud environments.  
    Flux runs in a single container, via docker-compose, virtual machines in the cloud, or easily in Kubernetes.    
    Elasticity and dynamism map well to auto-scaling and ephemeral resources. Programming a workload to use
    Flux thus makes it portable across environments.
```

Or different title:

```console
Flux is a portability layer
```